components/contribution-flow/index.js

```diff
@@ -483,12 +491,21 @@ class CreateOrderPage extends React.Component {
   }
 
   getLoggedInUserDefaultContibuteProfile() {
+    const { LoggedInUser } = this.props;
+    let profile = null;
+
     if (get(this.state, 'stepProfile')) {
-      return this.state.stepProfile;
+      profile = this.state.stepProfile;
+    } else if (LoggedInUser) {
+      profile = { email: LoggedInUser.email, image: LoggedInUser.image, ...LoggedInUser.collective };
     }
 
-    const { LoggedInUser } = this.props;
-    return !LoggedInUser ? null : { email: LoggedInUser.email, image: LoggedInUser.image, ...LoggedInUser.collective };
+    if (this.props.contributeAs) {
+      const [, profiles] = this.getProfiles();
```

Comment:
Seeing that, I'm not a fan of the return value of `getProfiles`, how could we make it clearer @Betree ?

Reply:
That was initially made to clearly separate your user profile from the others. Splitting in two helpers like `getUserProfile` and `getOtherProfiles` could do the job.

Reply:
@leonardohof @Betree Should we take the opportunity to do that now?